seedpod_ground_risk/pathfinding/theta_star.py

```diff
@@ -7,11 +7,21 @@
 from seedpod_ground_risk.pathfinding.a_star import _reconstruct_path
 from seedpod_ground_risk.pathfinding.algorithm import Algorithm
 from seedpod_ground_risk.pathfinding.environment import Node, GridEnvironment
+from seedpod_ground_risk.pathfinding.heuristic import Heuristic, ManhattanHeuristic
 
 
-class RiskThetaStar(Algorithm):
+class GridThetaStar(Algorithm):
```

Comment:
This class doesn't get used anywhere and does not abstract any functionality, so is useless